docs/_plugins/search_index.rb

```diff
@@ -201,6 +210,10 @@ def execute
   editions.add(edition_short) unless edition_short.empty?
   uniq_for_indexing << uniq if changed_filenames.include?(post.basename)
   changed_filenames.delete(post.basename)
+
+  key = model[:id]
+  name_edition_language_models_group[key] = [] unless name_edition_language_models_group.has_key? key
```

Comment:
Please rename this variable also.

Reply:
@pabla , Should I rename it to `name_language_editions_to_models_mapping` ?

Reply:
Yes, something like that. I mean the `name_edition_language` part should be in the right order.

Reply:
Done